src/data/reports/glass-data-submission/GLASSDataSubmissionDefaultRepository.ts

```diff
@@ -147,7 +149,13 @@ export class GLASSDataSubmissionDefaultRepository implements GLASSDataSubmission
             enrolledCountries.join(";"),
             modules
         );
-        const registrations = await this.getRegistrations(dataSubmissionItems);
+
+        const questionnaireDataSets =
```

Comment:
can you rename questionnaireDataSets to questionnaires as modules may have questionnaires that are program as well.